torch/_strobelight/examples/compile_time_profile_example.py

```diff
@@ -1,11 +1,10 @@
 # mypy: allow-untyped-defs
 import torch
-from torch._strobelight.compile_time_profiler import StrobelightCompileTimeProfiler
+from torch._utils_internal import enable_compiletime_strobelight
 
 
 if __name__ == "__main__":
-    # You can pass TORCH_COMPILE_STROBELIGHT=True instead.
```

Comment:
I guess StrobelightCompileTimeProfiler.enable() will work also that way it ended up being.

Reply:
Yes, that is correct. 

Reason to move to enable_* here to be consistent so that any future changes on enablement can be all handled inside the enable_ function.